8.x/theme-development/products.md

```diff
@@ -1,3 +1,435 @@
+|---.envrc
+|---.envrc.example
+|---.gitignore
+|---composer.json
+|---deploy.sh
+|---mix-manifest.json
+|---package.json
+|---package-lock.json
+|---webpack.mix.js
+```
+
+**TIP:** *Boilerplate is any text, documentation, or procedures that can be reused more than once in a new context without any substantial changes to the original.* *<sup>[1](#1)</sup>* 
+
+**TIP:** *The blade is the simple, yet powerful templating engine that is included with Laravel.*  *<sup>[2](#2)</sup>* 
+
+Presently if you enter the product's single page, **how can you show the item properties?**
+Larammerce will tell you.
+
+if you click on a product in the store, you will enter a single page with a dedicated URL that displays content from the blade called `product-single.blade.php`. The procedure, in general, is as which if you enter the `product-single.blade.php` file in the `public/views` path of the Boilerplate template, you can display the properties on the single product page by calling the properties.
+
+**NOTE:** The `product-single.blade.php` and `product-single_mobile.blade.php` pages are for displaying single product pages.
+
+**NOTE:** If you go to the `product-single.blade.php` file, you will notice that the file's `@extends (_'base')` was used first, which is because the site's headers and footers pages are similar, and it is not necessary to create a template for this page.
+
+For example, You delete the entire content of the page and write the following command in the terminal:
+
+##### EXAMPLE
+
+```bash
+./deploy.sh #to deploy the resources to the backend directory
+```
+Write the following code snippet:
+
+##### EXAMPLE
+
+```php
+<h1>Hello World!</h1>
+```
+
+And write again in the terminal:
+
+##### EXAMPLE
+
+```bash
+./deploy.sh 
+```
+
+Finally, enter the browser and after refreshing, you will see that the changes have been applied and you will see the phrase "Hello World".
+
+Presently that you simply know how it works, the Larammerce group will center on the most theme of this area, which is the show of properties.
+So, let's take some steps:
+
+**NOTE:** When you work on the `product-single.blade.php` file, the backend panel gives you a variable called `$product`. You can use it to view fields and product properties.
+
+##### SOURCE
+
+```php
+@php
+   dd($product); #view fields and product properties
+@endphp
+```
+
+**TIP:** *As you know, you can also use blade tags.* *<sup>[2](#2)</sup>* 
+
+**TIP:** *Blade's {{ }} echo statements are automatically sent through PHP's `htmlspecialchars` function to prevent XSS attacks.* *<sup>[2](#2)</sup>*
+
+And write in the terminal:
+
+```bash
+./deploy.sh 
+```
+Finally, go to the browser and refresh, you will see a product model and if you open the attributes field all the properties will be available to you. If you are not sure what properties to use, you can use this method.
+
+Now, you can display the product **title**, **product code**, **latest price**, **previous price**, and **Count**:
```

Comment:
Please be precious about the minor or capital cases of the words and code segments.
For example, **Code** should be changed to **code**.